microsite-next/sidebars.json

```diff
@@ -1 +1,366 @@
-{}
+{
+  "releases": {
+    "Release Notes": [
+      "releases/v1.8.0",
```

Comment:
I think we can live with this being slightly out of sync 😅

Reply:
Yeah it's good enough for now :)